tst/hooks/sdk/useActiveSpeakersState.test.tsx

```diff
@@ -1,7 +1,7 @@
 // Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
 // SPDX-License-Identifier: Apache-2.0
 
-import {LogLevel} from "amazon-chime-sdk-js";
+import {LogLevel} from 'amazon-chime-sdk-js';
```

Comment:
>describe('Icon Button', () => {

Can yiou help me fix the test name? :D

Reply:
I fixed it in my PR, Levi you can ignore this.